src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,85 @@
+import { useRef, useState } from "react";
+import BrandLogo from "../../assets/images/simplifylogo.svg";
+
+
+export const Navbar = () => {
+
+    const [isHidden, setHidden] = useState(true);
+
+    const navBarRef = useRef("-100%");
+
+    const openSideBar = (event) => {
+        event.stopPropagation();
+        navBarRef.current.style.right = "0";
+        setHidden(!isHidden);
+    }
+
+    const closeSideBar = (event) => {
+        event.stopPropagation();
+        navBarRef.current.style.right = "-100%";
+        setHidden(!isHidden);
+    }
+
+    const downloadFile = () => {
+        
+    }
+
+    return (<nav className="navbar">
+        <div className="container-xl">
+            <div className="nav-brand">
+                <img src={BrandLogo} alt="Brand Logo" width="30" height="30" />
+                <span className="nav-brand-title">implify</span>
+            </div>
+            <div className="nav-wrapper">
+                <div className="collapse navbar-collapse mob_none">
+                    <ul className="navbar-nav flex-center">
+                        <li className="nav-item"><a href="/">Home</a></li>
+                        <li className="nav-item"><a href="/documentation">Documentation</a></li>
+                        <li className="nav-item"><a href="/about">About</a></li>
```

Comment:
for desktop site can you add effect to navbar items to show its active.  something like you did on hover for documentation page.

Reply:
Actually this requires javaScript which i think is irrelevant for component Library i have added Styles for active classes though so it can be integrated with other react apps elsewhere.